www/artists/thall/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Why are you touching thall? Who is thall? thall doesn't have a page on the database. Is this supposed to be a new artist? If so, that should be a different pull request on a different branch. If not, then what is this?!?